src/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -6,24 +6,28 @@ import { ValidationErrors } from '@angular/forms';
 })
 export class FirstErrorPipe implements PipeTransform {
 
-  transform(value: ValidationErrors): string {
+  transform(value: ValidationErrors, args?: string): string {
     if (!value) {
       return '';
     }
 
+    if (!args) {
+      args = 'field';
+    }
+
     let keys = Object.keys(value);
```

Comment:
_Nitpick:_
`const`.

Reply:
Good spot, @paul-graham - damn... how did I miss that one? 😄

Reply:
changed to const